package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.6.2",
   "description": "Bundle references from an single AsyncAPI document into a single file.",
   "main": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "types": "lib/types.d.ts",
```

Comment:
This way user will not get the types support for the bundle function, I think it is better to export the types from `index.ts` file.